_posts/2017-08-28-upgrade-to-rails-3-1.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title:  "Upgrade Rails from 3.0 to 3.1"
+date: 2017-08-28 11:02:00
+categories: ["rails", "upgrades"]
+author: "luciano"
+---
+
+This is the second article of our [Upgrade Rails series](https://www.ombulabs.com/blog/tags/upgrades). We will be covering the most important aspects that you need to know to update your Ruby on Rails application from version 3.0 to 3.1. If you are in an older version, you can take a look at our [previous article](https://www.ombulabs.com/blog/rails/upgrades/upgrade-to-rails-3-rails-upgrades.html).
+
+<!--more-->
+
+1. Considerations
+2. Ruby version
+3. Config files
+4. jQuery
+5. Asset Pipeline
+6. Next steps
+
+
+## 1. Considerations
+Before beginning with the upgrade process, we recommend that each version of your Rails app has the latest [patch version](http://semver.org) before moving to the next major/minor version. For example, in order to follow this article, your [Rails version](https://rubygems.org/gems/rails/versions) should be at 3.0.20 before updating to Rails 3.1.12
+
+## 2. Ruby version
+Rails 3.1 requires Ruby 1.8.7 or higher, but no more than 1.9.3. If you want to use Ruby 1.9.x, we recommend you skip directly to 1.9.3. Also Ruby 1.9.1 is not usable because it has segmentation faults on Rails 3.1. That means that the compatible Ruby versions for Rails 3.1 are 1.8.7, 1.9.2, or 1.9.3. Keep in mind that the next Rails version (3.2) will be the last one that supports Ruby 1.8.7 and 1.9.2.
```

Comment:
@lubc Please link the ruby versions (e.g. Ruby 1.8.7) to something like https://www.ruby-lang.org/en/news/2008/05/31/ruby-1-8-7-has-been-released/